invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -433,3 +433,11 @@ dl.details-list {
   margin-left: auto;
 }
 
+.flex-container {
+  display: flex !important;
+}
+
+
+.contents-container {
```

Comment:
what is this class needed for ?

Reply:
I meant more what kind of a problem does it solve, sorry for not being specific. We noticed also yesterday that the elements need to be aligned vertically in the middle (filters are a bit misaligned with the search bar